src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -550,17 +549,18 @@ object ProtocPlugin extends AutoPlugin {
 
   def protocIncludeDependencies: Def.Initialize[Seq[File]] =
     Def.setting {
-      def filter =
-        ScopeFilter(inDependencies(ThisProject, includeRoot = false), inConfigurations(Compile))
       (
         PB.protoSources.?.all(filter).value.map(_.getOrElse(Nil)).flatten ++
           PB.includePaths.?.all(filter).value.map(_.getOrElse(Nil)).flatten
       ).distinct
     }
 
+  private[this] def filter: ScopeFilter =
+    ScopeFilter(inDependencies(ThisProject, includeRoot = false), inConfigurations(Compile))
```

Comment:
https://github.com/sbt/sbt/issues/1095#issuecomment-33209107 (looks like it was fixed in sbt 1.3 ...)